python/lsst/ts/scheduler/config_schema.py

```diff
@@ -116,440 +129,446 @@ class inside the module.
+                type: number
+                description: Minimum rotator position (degree).
+              maxpos:
+                type: number
+                description: Maximum rotator position (degree).
+                default: 90.0
+              filter_change_pos:
+                type: number
+                default: 0.0
+                description: Rotator position for filter changes (degree).
+              maxspeed:
+                type: number
+                description: Maximum rotator speed (degrees/s).
+                default: 3.5
+              accel:
+                type: number
+                description: Rotator acceleration (degrees/s^2).
+                default: 1.0
+              decel:
+                type: number
+                description: Rotator deceleration (degrees/s^2).
+                default: 1.0
+              follow_sky:
+                type: boolean
+                description: >-
+                  If True enables the movement of the rotator during slews to put
+                  North-Up. If range is insufficient, then the alignment is
+                  North-Down If the flag is False, then the rotator does not move
+                  during the slews, it is only tracking during the exposures. Note
+                  that this must be TRUE to allow *any* movement of the rotator
+                  during a slew. FALSE locks the rotator.
+                default: True
+              resume_angle:
+                type: boolean
+                description: >-
+                  If True enables the rotator to keep the image angle after a
+                  filter change, moving back the rotator to the previous angle
+                  after the rotator was placed in filter change position. If the
+                  flag is False, then the rotator is left in the filter change
+                  position. This must be TRUE to allow any movement of the
+                  rotator after a filter change.
+                default: true
+            additionalProperties: false
+          camera:
+            type: object
+            properties:
+              readout_time:
+                type: number
+                decription: Camera readout time (seconds).
+                default: 2.0
+              shutter_time:
+                type: number
+                description: Time it takes to open/close camera shutter (seconds).
+                default: 1.0
+              filter_change_time:
+                type: number
+                description: Time it takes to perform a filter change (seconds).
+                default: 120.0
+              filter_max_changes_burst_num:
+                type: number
+                description: Maximum number of filter changes in a night.
+                default: 1.
+              filter_max_changes_burst_time:
+                type: number
+                description: Minimum time (units=seconds) between filter changes in a night.
```

Comment:
In almost all other cases you omit `units=` when indicating the unit so please remove it here and also from line 367.